### DISCUSSION TITLE
Convince me: Migrate from Shopify Plus to Swell ($3-5 mill annual rev Store)

### DISCUSSION THREAD

**Discussion by aanomm** [swellstores/community#309] [Help] [answered]
Inspired by @DennisKraaijeveld's "Convince Me" thread ( #125 ) I'm pitching in a the same vein just with a different Store Revenue range and a different Shopify versions (Shopify Plus)...

I have a client in horrible Shopify situation: 

- They're stuck in the stock standard Shopify Debut theme (a 2010 theme!) 
- Their Recharge is a nightmare mess for everyone (their customers, Customer Support, and any Dev who dares delve into their dinosaur code mess)
- Their Shopify Products are a nightmare mess (1800 products with countless duplicates to workaround Recharge shortfalls)
- They are a Shopify Plus store with an annual revenue of ~ $3-$5 million, but they have very little understanding of websites and ecomm and don't even have web dev on staff (the Marketing Team have historically managed ecomm... which is why it's all such a mess)

I've been a fan of Swell from afar since Swell launched, though I've not yet had a chance to try Swell IRL. I know Swell would make parts of their ecomm life so much simpler, but there are so many unknowns with Swell and it feels like the migration to Swell would be such a mammoth task? 

They are open to the idea of repositioning their ecommerce, but they need something new online for November 1 (November is their biggest sales month of the year by far)

I'd love to hear your thoughts...

**Comment by ericingram** [comment, marked as answer]
This is a fun challenge - but more importantly it sounds like the merchant is suffering from the awkward subscription integration between Recharge and Shopify. A full migration might be daunting, although we have many ideas to ease the effort soon, today it requires a lot of work.

Still, there is a lot of upside to consider. Native subscriptions would be a huge improvement with Swell. And future growth opportunities can be enabled by faster development of new features with a fully customizable data model.

Beyond the current environment, Swell's focus is now on building an App ecosystem. Flexibility of the platform is going to be a game-changer for App developers and merchants of all kinds. You could argue, the earlier they adopt it, the further ahead they would be among competitors.

I know it sounds like a long-shot, but we've been building toward this goal for over a decade now, and we're *super* close to nirvana.
